src/main/java/org/dspace/pack/bagit/BagItPolicyUtil.java

```diff
@@ -0,0 +1,236 @@
+            final ResourcePolicy resourcePolicy = resourcePolicyService.create(Curator.curationContext());
+
+            final String rpName = policy.getName();
+            if (rpName != null) {
+                resourcePolicy.setRpName(rpName);
+            }
+
+            final String rpDescription = policy.getDescription();
+            if (rpDescription != null) {
+                resourcePolicy.setRpDescription(rpDescription);
+            }
+
+            final String rpStartDate = policy.getStartDate();
+            if (rpStartDate != null) {
+                try {
+                    final Date date = dateFormat.parse(rpStartDate);
+                    resourcePolicy.setStartDate(date);
+                } catch (ParseException ignored) {
+                    logger.warn("Failed to parse rp-start-date. The date needs to be in the format 'yyyy-MM-dd'.");
+                }
+            }
+
+            final String rpEndDate = policy.getEndDate();
+            if (rpEndDate != null) {
+                try {
+                    final Date date = dateFormat.parse(rpEndDate);
+                    resourcePolicy.setEndDate(date);
+                } catch (ParseException ignored) {
+                    logger.warn("Failed to parse rp-end-date. The date needs to be in the format 'yyyy-MM-dd'.");
+                }
+            }
+
+            final String groupName = policy.getGroup();
+            final String epersonEmail = policy.getEperson();
+            if (groupName != null) {
+                final String nameForImport;
+
+                if (groupName.equalsIgnoreCase(Group.ADMIN) || groupName.equalsIgnoreCase(Group.ANONYMOUS)) {
+                    nameForImport = groupName;
+                } else {
+                    nameForImport = PackageUtils.translateGroupNameForImport(Curator.curationContext(), groupName);
+                }
+
+                final Group group = groupService.findByName(Curator.curationContext(), nameForImport);
+                if (group == null) {
+                    throw new PackageException("Could not find group " + nameForImport + " in the database! If this" +
+                                               "is either the ADMIN or ANONYMOUS group check that your database is" +
+                                               "initialized correctly.");
+                }
+
+                resourcePolicy.setGroup(group);
+            } else if (epersonEmail != null) {
+                final EPerson ePerson = ePersonService.findByEmail(Curator.curationContext(), epersonEmail);
+                if (ePerson == null) {
+                    throw new PackageException("Could not find ePerson " + epersonEmail + " in the database!");
+                }
+
+                resourcePolicy.setEPerson(ePerson);
+            } else {
+                // throw an exception as well?
+                logger.warn("Cannot import policy, no rp-group or rp-eperson attribute found on value!");
```

Comment:
Wasn't quite sure how best to handle this -- if the policy.xml does not contain a group or an eperson. Looking at the METSRightsCrosswalk it seems to log an error.